saleor/payment/gateways/np_atobarai/api_helpers.py

```diff
@@ -0,0 +1,258 @@
+import logging
+from decimal import Decimal
+from typing import TYPE_CHECKING, Dict, Iterable, List, Optional
+
+import requests
+from django.utils import timezone
+from posuto import Posuto
+from requests.auth import HTTPBasicAuth
+
+from saleor.order.models import Order
+
+from ...interface import AddressData, PaymentData
+from ...utils import price_to_minor_unit
+from .api_types import NPResponse, error_np_response
+from .const import NP_ATOBARAI, NP_TEST_URL, NP_URL, REQUEST_TIMEOUT
+from .errors import (
+    BILLING_ADDRESS_INVALID,
+    NO_BILLING_ADDRESS,
+    NO_PSP_REFERENCE,
+    NO_SHIPPING_ADDRESS,
+    NO_TRACKING_NUMBER,
+    NP_CONNECTION_ERROR,
+    SHIPPING_ADDRESS_INVALID,
+    SHIPPING_COMPANY_CODE_INVALID,
+)
+from .utils import notify_dashboard, np_atobarai_opentracing_trace
+
+if TYPE_CHECKING:
+    from . import ApiConfig
+
+
+logger = logging.getLogger(__name__)
+
+
+def get_url(config: "ApiConfig", path: str = "") -> str:
+    """Resolve test/production URLs based on the api config."""
+    return f"{NP_TEST_URL if config.test_mode else NP_URL}{path}"
```

Comment:
Wouldn't be better to move URLs to settings and get it directly depends on current environment? Checking environment each call isn't good idea tbh